tomopy/io/exchange.py

```diff
@@ -63,12 +62,11 @@
 logger = logging.getLogger(__name__)
 
 
-__author__ = "Doga Gursoy, Luis Barroso-Luque"
+__author__ = "Doga Gursoy"
 __credits__ = "Francesco De Carlo"
 __copyright__ = "Copyright (c) 2015, UChicago Argonne, LLC."
 __docformat__ = 'restructuredtext en'
 __all__ = ['read_als_832',
-           'read_als_832h5',
```

Comment:
I think my version was not fully up-to-date. There is also some changes in Petra that i did not want to make.